docs/user-guide/postcss-plugin.md

```diff
@@ -108,6 +108,30 @@ gulp.task("build:scss", function () {
 
 ### Example D
 
+Using the plugin with [`gulp-postcss`](https://github.com/postcss/gulp-postcss) and [`postcss-less`](https://github.com/webschik/postcss-less) to lint Less, and as part of the build task:
```

Comment:
I find it kind of confusing to have nearly identical examples here and above in which the only real difference is the replacement of `syntax: "scss"` with `syntax: "less"`. If that's only thing to point out — that `"less"` can be used as well as `"scss"` — then I say we just point that out and leave it, or even make `"less"` the primary example in at least one of these cases. But I don't think we should just duplicate the examples for every additional syntax we add.